axelor-human-resource/src/main/resources/views/TimesheetLine.xml

```diff
@@ -257,5 +333,15 @@
   		<context if="id != null &amp;&amp; !isShowPhasesElements" name="_projectIds" expr="eval:[id]"/>
   		<context if="id != null &amp;&amp; isShowPhasesElements" name="_projectIds" expr="eval:[id] + childProjectList.collect{it->it.id}"/>
 	</action-view>
+	
+	<action-view name="action-teamtask-dashlet-validated-timeshet-lines" model="com.axelor.apps.hr.db.TimesheetLine" title="Validated Timesheet lines">
+		<view type="grid" name="timesheet-line-teamtask-grid"/>
```

Comment:
add view for type="form"

Reply:
In form view can i have to show teamtask?? bcoz we hide it in grid view as said in the requirement.

Reply:
for the moment you can use the existing form view without hiding teamTask

Reply:
when we click on timsheetline grid in time spent tab in team task it will open new tab and show the form view of the timesheetline is it okay ??